components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/applications/add.js

```diff
@@ -116,10 +116,30 @@ var addApplication = function () {
                                 }
                             ]
                         });
+                    } else if (jsonPayload != null && jsonPayload != "") {
```

Comment:
Better to use if(!jsonPayload) for null checks rather than using if(jsonPayload != null)